src/hooks/useUniswapPools.js

```diff
@@ -189,7 +189,7 @@ export default function useUniswapPools(sortField, sortDirection, token) {
   );
 
   const handleGetUniswapV2PoolsResponse = useCallback(() => {
-    const { pools } = poolData.data;
+    const { pools } = poolData;
```

Comment:
according to the types definition, `getUniswapV2Pool` returns `UniswapPoolData[]` not an object with `pools` inside?